src/components/capture/creation/index.tsx

```diff
@@ -58,48 +70,40 @@ function NewCaptureModal() {
 
     // Form Event Handlers
     const handlers = {
-        close: () => {
-            navigate('..'); //This is assuming this is a child of the /captures route.
-        },
-
-        delete: () => {
-            console.log('Delete? You sure?');
-        },
-
-        save: (event: MouseEvent<HTMLElement>) => {
+        addToChangeSet: (event: MouseEvent<HTMLElement>) => {
             event.preventDefault();
-            let catalogVal = '';
 
-            if (editorRef.current) {
-                catalogVal = editorRef.current.getValue();
-            }
+            const namespace: string = details.data.name;
```

Comment:
I think we should use a different name for this variable given namespace has meaning in typescript.

Reply:
Pending correction. Agreed. I arrived at that conclusion shortly after opening this PR. It was meant to be a draft to review and correct any mistakes before anyone else took a critical look, but I forget to press that button yesterday evening.